cmd/kube-controller-manager/app/core.go

```diff
@@ -575,8 +575,7 @@ func startPVCProtectionController(ctx ControllerContext) (controller.Interface,
 		ctx.InformerFactory.Core().V1().PersistentVolumeClaims(),
 		ctx.InformerFactory.Core().V1().Pods(),
 		ctx.ClientBuilder.ClientOrDie("pvc-protection-controller"),
-		utilfeature.DefaultFeatureGate.Enabled(features.StorageObjectInUseProtection),
```

Comment:
It looks like there was a bug here. cc @pohly 
Can you make sure to fix it in older versions?

Reply:
Would it be possible to first submit a PR with just this bug fix, then rebase this PR on top of it?

As it stands now, the change note of this PR doesn't document that it fixes a bug and the usual backport request procedure won't work.

Reply:
I hope the PR #104911 will help with this.

Reply:
I created one: https://github.com/kubernetes/kubernetes/pull/104913

Reply:
Oh. Then I'll close mine.